src/dmd/globals.d

```diff
@@ -257,11 +261,11 @@ extern (C++) struct Param
     uint versionlevel;                  // version level
     Array!(const(char)*)* versionids;   // version identifiers
 
-    const(char)[] defaultlibname;        // default library for non-debug builds
-    const(char)[] debuglibname;          // default library for debug builds
-    const(char)[] mscrtlib;              // MS C runtime library
+    const(char)[] defaultlibname;       // default library for non-debug builds
+    const(char)[] debuglibname;         // default library for debug builds
+    const(char)[] mscrtlib;             // MS C runtime library
 
-    const(char)[] moduleDepsFile;        // filename for deps output
+    const(char)[] moduleDepsFile;       // filename for deps output
```

Comment:
As much as my OCD would like to see this aligned, please do it in a separate PR.

Reply:
Undone